cli/gardener_ci/compliance.py

```diff
@@ -48,13 +48,14 @@ class DiffArguments:
     right_name: str
     left_version: str
     right_version: str
+    outfile_prefix: str
     ocm_repo_urls: list[str]
     exclude_component_names: list[str] = None
     exclude_component_resource_names: list[ComponentResourceNames] = None
     resource_types: list[str] = None
     name_template: str = None
     name_template_expr: str = None
-    outfile_prefix: str = 'resource-diff'
+    
```

Comment:
I think this is one line too much (pep8 and our linter mandates exactly two lines - see linter output (linked to your pullrequest))